lmdeploy/vl/templates.py

```diff
@@ -303,6 +303,11 @@ def get_vl_prompt_template(model_path: str, chat_template: BaseModel,
         return QwenVLChatTemplateWrapper(chat_template)
     elif arch in ['LlavaLlamaForCausalLM', 'LlavaMistralForCausalLM']:
         return LlavaVLChatTemplateWrapper(chat_template)
+    elif arch in [
```

Comment:
It can merge with branch `elif arch in ['LlavaLlamaForCausalLM', 'LlavaMistralForCausalLM']:`

Reply:
ok